src/services/data-provider.js

```diff
@@ -74,18 +82,22 @@ export default {
     return httpClient(endpoint.url, {
       method: "POST",
       body: JSON.stringify(params.data),
-    }).then(({ json }) => ({
-      data: endpoint.getData(json.data),
-    }));
+    })
+      .then(({ json }) => ({
+        data: endpoint.getData(json.data),
+      }))
+      .catch(Promise.reject);
   },
 
   delete: (resource, params) => {
     const endpoint = endpoints(DELETE, resource, params);
     httpClient(endpoint.url, {
       method: "DELETE",
-    }).then(({ json }) => ({
-      data: endpoint.getData(json.data),
-    }));
+    })
+      .then(({ json }) => ({
+        data: endpoint.getData(json.data),
+      }))
+      .catch(Promise.reject);
   },
 
   deleteMany: (resource, params) => {
```

Comment:
not being triggered
![image](https://user-images.githubusercontent.com/17856665/88460970-0c9a2180-ce98-11ea-94d4-9d9c06b4aa45.png)

Reply:
I have removed these.